src/elements/List/ListList.js

```diff
@@ -0,0 +1,42 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import {
+  customPropTypes,
+  getElementType,
+  getUnhandledProps,
+  META,
+  useKeyOnly,
+} from '../../lib'
+
+function ListList(props) {
+  const { children, className } = props
+
+  const ElementType = getElementType(ListList, props)
+  const classes = cx(
+    className,
+    useKeyOnly(ElementType !== 'ul' && ElementType !== 'ol', 'list'),
```

Comment:
> Per the docs the "list" class may be omitted "as a convenience" when using an "ol" or "ul" tag.

Sad, but it must be omitted.

Without classes:
![image](https://cloud.githubusercontent.com/assets/14183168/18981851/68d4869c-86ea-11e6-8e8e-5d6693609ec1.png)
With classes:
![image](https://cloud.githubusercontent.com/assets/14183168/18981899/beca1030-86ea-11e6-88fa-6c1cbde2659b.png)